wallet/src/Pos/Wallet/Web/State/State.hs

```diff
@@ -482,7 +506,7 @@ flushWalletStorage :: (MonadIO m, HasConfiguration)
                    -> m ()
 flushWalletStorage = updateDisk A.FlushWalletStorage
 
-applyModifierToWallet
+applyModifierToWallet2
```

Comment:
Very interested to hear about this bit. Essentially, I was concerned that `applyModifierToWallet` could still have been in the transaction log, so I have added another acidic transaction (see below) and stopped exporting the original `applyModifierToWallet`.